lib/fathead/perl_doc/cover/functions.txt

```diff
@@ -0,0 +1,275 @@
+-A
```

Comment:
Yep, you find an independent source for keywords, add them to the cover/ directory.

They're split into entities like functions, operators etc. in order to test for or perhaps generate a common set of redirects.